README.md

```diff
@@ -149,6 +158,36 @@ Here are a selected few results from various experiments we conducted. Our exper
 <sub> w/ learning rate=9e-06, max train steps=200 (<a href="https://huggingface.co/datasets/chansung/me">datasets</a> | <a href="https://wandb.ai/chansung18/dreambooth-generate-me?workspace=user-chansung18">reports</a>)</sub>
 </div><br>
 
+## Using in Diffusers 🧨
+
+The [`diffusers` library](https://github.com/huggingface/diffusers/) provides state-of-the-art tooling for experimenting with
+different Diffusion models, including Stable Diffusion. It includes 
+different optimization techniques that can be leveraged to perform efficient inference
+with `diffusers` when using large Stable Diffusion checkpoints. One particularly 
+advantageous feature `diffusers` has is its support for [different schedulers](https://huggingface.co/docs/diffusers/using-diffusers/schedulers) that can
+be configured during runtime and can be integrated into any compatible Diffusion model.
+
+Once you have obtained the DreamBooth fine-tuned checkpoints using this codebase, you can actually
+export those into a handy [`StableDiffusionPipeline`](https://huggingface.co/docs/diffusers/api/pipelines/stable_diffusion/overview) and use it from the `diffusers` library directly. 
+
+Consider this repository: [chansung/dreambooth-dog](https://huggingface.co/chansung/dreambooth-dog). You can use the
+checkpoints of this repository in a `StableDiffusionPipeline` after running some small steps:
+
+```py
+from diffusers import StableDiffusionPipeline
+
+model_ckpt = "sayakpaul/text-unet-dogs-kerascv_sd_diffusers_pipeline"
```

Comment:
I thought readers could be confused a bit since the paragraph above says "Consider this repository: chansung/dreambooth-dog" which is different. If you think it is OK, you can go ahead and merge the PR(I mark this PR as approved)!

```suggestion
# checkpoint of the converted Stable Diffusion from KerasCV
model_ckpt = "sayakpaul/text-unet-dogs-kerascv_sd_diffusers_pipeline"
```